dressForTheWeather/View Controllers/RecommendationViewController.swift

```diff
@@ -9,42 +9,54 @@
 import UIKit
 
 final class RecommendationViewController: UIViewController {
-    
+
```

Comment:
These empty space commits can be reverted

Reply:
I prefer the space around the MARK notation for readability. It's consistent with how I have those spaced in other files, so I'm going to leave it for now.